src/d3d11/d3d11_device.cpp

```diff
@@ -2260,6 +2258,46 @@ namespace dxvk {
   }
 
 
+  HRESULT D3D11Device::OpenSharedResourceGeneric(
+      BOOL        nt,
```

Comment:
`nt` should a) have a more descriptive name and b) perhaps be a template parameter instead, so we have more options to use things like `if constexpr`.

Reply:
Unfortunately I'm not familiar enough with c++ to know how the template suggestion would work.